### PR TITLE
feat: pull gateway_mfr build from upstream to reduce build time

### PR DESCRIPTION
**Pull request by shawaj**
**Issue**

- Link: #43 
- Summary: pull pre-built binary from upstream to reduce build time

**How**

- pull gateway_mfr build from upstream
- update pypi and test-pypi actions
- add action to check gateway-mfr-rs version weekly and auto-update if necessary
- remove old unused backup action
- bumping minor version as this is a fairly substantial change to the build process
- update README to add more release info and update ROCK Pi hardware info

**Screenshots**

N/A

**References**

Closes: #43
Closes: #36 
Related to: #20 

Draft until upstream PR merged https://github.com/helium/gateway-mfr-rs/pull/10 and `gateway-mfr-v0.1.6-arm-unknown-linux-gnueabihf.tar.gz` is available

**Checklist**

- [x] Tests added
- [x] Cleaned up commit history (rebase!)
- [x] Documentation added
- [x] Thought about variable and method names

### CODE REVIEW

**Comment by shawaj** [comment]
Need to bump version number before merge, and also re-run the testpypi action once the upstream file is available

File: .github/workflows/update-gateway-mfr-rs.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          draft: false
+      - name: Report Status
+        if: env.UPDATED == 'true'
+        uses: ravsamhq/notify-slack-action@master
+        with:
+          status: ${{ job.status }}
+          notification_title: 'Gateway-mfr-rs has been updated to ${{ env.GATEWAY_MFR_RS_RELEASE }}. Please review PR!'
+          message_format: '{emoji} *{workflow}* {status_message} in <{repo_url}|{repo}>'
+          footer: '<{run_url}|View Run> | Linked Repo <{repo_url}|{repo}> | <{workflow_url}|View Workflow>'
+          mention_groups: 'S02GCFWL27R'
```
Comment (marvinmarnold) [inline .github/workflows/update-gateway-mfr-rs.yml:64]: Should this be an env var too?

**Review by marvinmarnold — APPROVED** [review]

File: .github/workflows/update-gateway-mfr-rs.yml (hunk 1 above)
Comment (shawaj) [reply .github/workflows/update-gateway-mfr-rs.yml:64]: The slack group ID you mean @marvinmarnold ?

I'm a little bit confused as to what you mean

Comment (marvinmarnold) [reply .github/workflows/update-gateway-mfr-rs.yml:64]: I mean replace `'S02GCFWL27R'` with `env.SLACK_MENTION_GROUPS` so that it isn't hardcoded.

Comment (shawaj) [reply .github/workflows/update-gateway-mfr-rs.yml]: OK done :-)

**Comment by shawaj** [comment]
Merging with failing build, because I want to test the update workflow correctly updates to the new v0.1.7 release just pushed today https://github.com/helium/gateway-mfr-rs/releases/tag/v0.1.7

**Pull request merged**